frame/composable-maths/src/dex/constant_product.rs

```diff
@@ -11,7 +11,7 @@ use sp_runtime::{
 	ArithmeticError, PerThing,
 };
 
-/// From https://balancer.fi/whitepaper.pdf, equation (2)
+/// From <https://balancer.fi/whitepaper.pdf>, equation (2)
```

Comment:
No, it was `cargo doc`. When running it, we get warnings about "raw links". The warning message advises enclosing the links in `<>` so that it becomes clickable.